### PR TITLE
Enables verb for observers to chemical/reagent scan.

### PR DESCRIPTION
**Pull request by Timberpoes**
## About The Pull Request

Adds the chemical scan from health analyzers as a verb for 2spooky observers to use.

New verb
image

Chem scan on its own
image

Chem scan alongside health scan
image

## Why It's Good For The Game

image

The people ask, and the people receive!

## Changelog
:cl:
add: Enabled new button allowing observers the ability to see chemicals and reagents identical to the health analyzer's chem scan functionality.
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
Could just tag it on to health scan verb

**Comment by Timberpoes** [comment]
> 
> 
> Could just tag it on to health scan verb

I was thinking that, but in testing the health scan is just so fucking spammy. It felt better to have them separately toggleable so people had the choice as to exactly what spammy crap they'd see on examining.

I also tested making health scan tri-state, so you'd toggle between health, chem and none but again, it felt a bit clunky to use.

File: code/modules/mob/dead/observer/observer.dm [hunk 1]
```diff
@@ -32,6 +32,7 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 	var/ghost_hud_enabled = 1 //did this ghost disable the on-screen HUD?
 	var/data_huds_on = 0 //Are data HUDs currently enabled?
 	var/health_scan = FALSE //Are health scans currently enabled?
+	var/chem_scan = FALSE //Are chem scans currently enabled?
 	var/gas_scan = FALSE //Are gas scans currently enabled?
```
Comment (spookydonut) [inline code/modules/mob/dead/observer/observer.dm:36]: these could be made flags but meh
